open-api/rest-catalog-open-api.yaml

```diff
@@ -1027,10 +1028,8 @@ components:
           type: integer
         summary:
           type: object
-          required:
-            - summary
```

Comment:
The operation is actually required, so this should just change to `operation` instead.

Reply:
Added operation as required 👍